apps/home/test/integration.coffee

```diff
@@ -2,16 +2,16 @@ servers = require '../../../test/helpers/servers'
 Browser = null
 sinon = require 'sinon'
 
-describe 'Homepage', ->
-  
+xdescribe 'Homepage', ->
```

Comment:
Yeah you can see the only test in this suite was pended already, so might as well do the whole suite at that point to avoid setup/teardown of the server. After Force + MG merger I'd like to take a stab at writing brittle/pend-inviting integrations tests—unfortunately most of them are pended at this point though.